### ISSUE TITLE
`.github/workflows/e2e.yaml` is misconfigured

### ISSUE THREAD

**Issue by joneshf** [crossplane-contrib/provider-keycloak#202]
I'm not entirely sure what's going on with https://github.com/crossplane-contrib/provider-keycloak/blob/5cd3042a585b7effa0ff708532e620d17c1c4892/.github/workflows/e2e.yaml, but it's running workflows when you comment on an issue. The workflows all appear to be failing: https://github.com/crossplane-contrib/provider-keycloak/actions/workflows/e2e.yaml?query=is%3Afailure.

More to the point, when a workflow fails, it will notify you however an individual has GitHub setup for workflow failures. E.g. when [this workflow failed](https://github.com/crossplane-contrib/provider-keycloak/actions/runs/12537040980), GitHub sent me an email about it. That's not really a notification I need to receive.

I'd toss up a PR, but I'm not sure if this problem can be fixed with code changes or if it's a setting in the repository/organization. From [the linked documentation](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#access-to-reusable-workflows), it looks like it might be a permissions issue. Or maybe it doesn't follow repository rename redirects? Maybe something else?

Once it does start running, it might be worth filtering to [only running on PR comments](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#issue_comment-on-issues-only-or-pull-requests-only) (rather than both issue comments and PR comments). Only saying that since it looks like that workflow is supposed to be for PR comments.

**Comment by Breee** [comment]
yeah i never understood why and i also cannot see the secrets as it's up to the admins to fix that. 
i'll just remove that stuff for now
